app/templates/my_dashboard/_my_dashboard_tabs.hbs

```diff
@@ -0,0 +1,41 @@
+<div class="row tabs tab-icons">
+  {{#link-to 'my_updates' tagName="div" classNames="small-4 updates tab columns text-center"}}
+    {{#link-to 'my_updates' }}
+      <i class="dashboard-menu">{{fa-icon 'bell-o'}}</i>
+    {{/link-to}}
+  {{/link-to}}
+
+  {{#link-to 'my_orders' tagName="div" classNames="small-4 orders tab columns text-center"}}
+    {{#link-to 'my_orders' }}
+      <i class="dashboard-menu">{{fa-icon 'calendar'}}</i>
+    {{/link-to}}
+  {{/link-to}}
+
+  {{#link-to 'my_account' tagName="div" classNames="small-4 account tab columns text-center"}}
+    {{#link-to 'my_account' }}
+      <i class="dashboard-menu">{{fa-icon 'vcard'}}</i>
+    {{/link-to}}
+  {{/link-to}}
+</div>
+
+<div class="row tabs tab-name active-tab">
+  <div class="small-4 columns updates tab text-center">
+  </div>
+  {{#link-to 'my_updates' tagName="div" classNames="small-4 updates tab active-tab columns text-center"}}
```

Comment:
how come there are 2 `link-to` within each other ?

Reply:
The above one is a `div` and lower one will become an anchor tag. 
cc - @namrataukirde

Reply:
Can we just replace it with a `<div>` in that case, having 2 links embedded like that doesn't make much sense to me